src/FixedSizeArray.jl

```diff
@@ -312,8 +313,16 @@ end
 
 parent_type(::Type{<:FixedSizeArray{<:Any,<:Any,Mem}}) where {Mem} = Mem
 
-underlying_storage(m) = m
-underlying_storage(f::FixedSizeArray) = f.mem
+"""
+    underlying_storage(f::FixedSizeArray)
+
+Return the underlying storage used by `f`.
+"""
+underlying_storage(f::FixedSizeArray) = getfield(f, :mem)
+
+# Internal function, not to have a method of the public `underlying_storage` on `::Any`.
+_underlying_storage(m) = m
+_underlying_storage(f::FixedSizeArray) = underlying_storage(f)
```

Comment:
I'm not convinced a public function which should only operate on `FixedSizeArray` should have a method on `::Any`, so I renamed this function which is used only internally to implement `copyto!`.